### PR TITLE
[CF-333] Extend functional tests to verify migration of config_drive VM attribute

### PR DESCRIPTION
**Pull request by jaspreetrawel**
Below scenario is added to the functional tests.

* Create VM with config_drive attribute in source cloud;
* Run migration
* Verify config_drive attribute migrated successfully

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: cloudferry_devlab/cloudferry_devlab/tests/config.py [hunk 1]
```diff
@@ -586,6 +586,8 @@
     {'name': 'server4', 'image': 'deleted_image', 'flavor': 'flavorname2'},
     {'name': 'server5', 'image': 'image1', 'flavor': 'flavorname1'},
     {'name': 'not_in_filter', 'image': 'image1', 'flavor': 'flavorname1'},
+    {'name': 'with_config_drive', 'image': 'image1', 'flavor': 'flavorname1',
+     'config_drive': True},
```
Comment (plomakin) [inline cloudferry_devlab/cloudferry_devlab/tests/config.py]: I think you can add this parameter to another VM which described above or below, increasing amount of VMs is not necessary for this case and also increasing load on devlab and jobs's execution time.

**Comment by plomakin** [comment]
This step:
- Verify config_drive attribute migrated successfully
has already covered in "test_migrate_vms_parameters" test case. So just fix comment above and it will be good.

**Comment by jaspreetrawel** [comment]
Yes the test case was already there. I just added the config and minute change in generate load. But time went in manually doing the hands-on for the config_drive feature and verifying how it works and after migration whether the metadata got migrated or not.

File: cloudferry_devlab/cloudferry_devlab/tests/config.py (hunk 1 above)
Comment (jaspreetrawel) [inline cloudferry_devlab/cloudferry_devlab/tests/config.py]: Sure, done.

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by roman-verchikov** [comment]
+1

**Comment by plomakin** [comment]
+1, can be merged

**Comment by jaspreetrawel** [comment]
Thanks @roman-verchikov @plomakin

**Pull request merged**